docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -2468,6 +2468,19 @@ definitions:
       authorization:
         $ref: '#/definitions/EventTypeAuthorization'
 
+      event_audience:
+        type: string
+        x-extensible-enum:
+          - component_internal
+          - business_unit_internal
+          - company_internal
+          - external_partner
+          - external_public
+        description: |
+          Zalando's internal detail. Intended target audience of the API. Relevant for standards around quality of
```

Comment:
I think that 'Zalando internal' should not end up in an open source project. Isn't it the case that it could be useful to someone else?

Reply:
I agree with you. I can delete this sentence and pretend it's useful to someone.

Reply:
0f0ee73